sycl/source/context.cpp

```diff
@@ -132,5 +133,32 @@ const property_list &context::getPropList() const {
   return impl->getPropList();
 }
 
+sycl::ext::oneapi::experimental::memory_pool
+context::ext_oneapi_get_default_memory_pool(const device &dev,
+                                            const usm::alloc &kind) const {
+
+  if (kind == sycl::usm::alloc::host) {
+    throw sycl::exception(
+        sycl::make_error_code(sycl::errc::invalid),
+        "Default host memory pool requested but device supplied!");
+  }
+
+  if (kind != sycl::usm::alloc::device)
```

Comment:
nit: this `if` doesn't have braces but the one just above does, we should make them both the same